worlds/AutoWorld.py

```diff
@@ -312,6 +319,15 @@ def remove(self, state: "CollectionState", item: "Item") -> bool:
     def create_filler(self) -> "Item":
         return self.create_item(self.get_filler_item_name())
 
+    @classmethod
+    def datapackage_checksum(cls) -> str:
+        datapackage = {
+            "items": cls.item_name_to_id,
+            "locations": cls.location_name_to_id,
+        }
+        dump = json.dumps(datapackage)
```

Comment:
I'm wondering if this should be NetUtils.encode, which puts out a minified representation, to get a "stable" output.

The default of JSON.stringify (i.e. space=null) and nlohman::json::dump (i.e. indent=-1) would be identical to our NetUtils.encode.